### PR TITLE
Small updates to versioning policy

### PR DESCRIPTION
**Pull request by sophiebits**

### CODE REVIEW

File: content/docs/faq-versioning.md [hunk 1]
```diff
@@ -41,6 +43,7 @@ In general, we *don't* bump the major version number for changes to:
 * **Development warnings.** Since these don't affect production behavior, we may add new warnings or modify existing warnings in between major versions. In fact, this is what allows us to reliably warn about upcoming breaking changes.
 * **APIs starting with `unstable_`.** These are provided as experimental features whose APIs we are not yet confident in. By releasing these with an `unstable_` prefix, we can iterate faster and get to a stable API sooner.
 * **Alpha and canary versions of React.** We provide alpha versions of React as a way to test new features early, but we need the flexibility to make changes based on what we learn in the alpha period. If you use these versions, note that APIs may change before the stable release.
+* **`react-reconciler` API changes.** The `react-reconciler` package is an experimental way to let you make your own renderers like React DOM itself. It uses the same version numbers as `react-dom`, but its API is not stable and may change at any time. If you depend on it, we recommend pinning to an exact version.
```
Comment (gaearon) [inline content/docs/faq-versioning.md]: Maybe we can just say this is the case for all 0.x releases?

File: content/docs/faq-versioning.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ Breaking changes are inconvenient for everyone, so we try to minimize the number
 
 Instead, we release new features in minor versions. That means that minor releases are often more interesting and compelling than majors, despite their unassuming name.
 
+Minor and patch releases are backwards compatible and won't have breaking changes. Backwards compatible means that if you take an existing app and upgrade to a new minor or patch release, we expect your code will still work without changes. (However, to use new features, you might need to restructure your code or wait for third-party library updates.)
```
Comment (gaearon) [inline content/docs/faq-versioning.md]: Should we mention the tension between fixing bugs and breaking changes?

**Comment by threepointone** [comment]
What should we do here next?

**Comment by sophiebits** [comment]
meh

**Pull request closed without merging**